### PR TITLE
add support for hp pavilion gaming 15 lid switch

### PR DESCRIPTION
**Pull request by bohanubis**
modified:   hwdb.d/60-keyboard.hwdb
first of all hi and this pull request is related to https://github.com/systemd/systemd/issues/28942
since even with the kernel update hopefully in 6.7 here https://lore.kernel.org/linux-acpi/20230920130506.8595-1-hdegoede@redhat.com/ made by jwrdegoede (I dont think its appropriate to tag him maybe I should I don't know how developers interact) (like I basically built the kernel with his patch)
the d7 key is still interpreted as a wlan key switch which should be unkown so yeah this commit will hopefully fix the issue
+ I just copied what I have in /etc/udev/hwdb.d/61-keyboard-local.hwdb into 60-keyboard-local.hwdb
I didn't compile systemd since I have no idea how to make a package out of it in arch so I guess that's enough

and well I guess the only possible problem with this pull request is 
`evdev:atkbd:dmi:bvn*:bvr*:bd*:br*:efr*:svnHP:pnHPPavilionGamingLaptop15-dk1*:*`
not being general enough or being too general in the other hand
this is the raw dmi thingy
`dmi:bvnInsyde:bvrF.45:bd07/07/2023:br15.45:efr2.51:svnHP:pnHPPavilionGamingLaptop15-dk1xxx:pvrType1ProductConfigId:rvnHP:rn8742:rvr02.51:cvnHP:ct10:cvrChassisVersion:sku20T68EA#BH4::`

### CODE REVIEW

File: hwdb.d/60-keyboard.hwdb [hunk 1]
```diff
@@ -609,6 +609,12 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pn*[pP][aA][vV][iI][lL][iI][oO
  KEYBOARD_KEY_d9=unknown
  KEYBOARD_KEY_d2=sysrq                                  # Fn+Print = SYSRQ
 
+
```
Comment (bluca) [inline hwdb.d/60-keyboard.hwdb]: extra newline

Comment (bohanubis) [reply hwdb.d/60-keyboard.hwdb]: done I guess (sorry doing my first ever pull request)

File: hwdb.d/60-keyboard.hwdb [hunk 2]
```diff
@@ -609,6 +609,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pn*[pP][aA][vV][iI][lL][iI][oO
  KEYBOARD_KEY_d9=unknown
  KEYBOARD_KEY_d2=sysrq                                  # Fn+Print = SYSRQ
 
+# Pavilion gaming 15-dk1* (lid keys)
+evdev:atkbd:dmi:bvn*:bvr*:bd*:br*:efr*:svnHP:pnHPPavilionGamingLaptop15-dk1*:*
+  KEYBOARD_KEY_d7=unknown
+  KEYBOARD_KEY_d8=unknown
```
Comment (bluca) [inline hwdb.d/60-keyboard.hwdb]: you have extra spaces, that is breaking the parser - each line here needs to be indented by one space only

Comment (bohanubis) [reply hwdb.d/60-keyboard.hwdb]: hello so I guess its fixed now can you review

**Review by bluca — APPROVED** [review]

**Pull request merged**